SwiftWisdom/Core/StandardLibrary/Numbers/UnsignedInteger+Extensions.swift

```diff
@@ -24,14 +16,38 @@ public func random(inRange range: CountableClosedRange<Int>) -> Int {
     return random
 }
 
-// TODO: write tests for this extension
 extension UnsignedInteger {
 
     public init(ip_data: Data) {
         let hexInt = ip_data.ip_hexInt ?? 0
         self.init(ip_safely: hexInt)
     }
-
+    
+    public init<T : SignedInteger>(ip_safely value: T) {
```

Comment:
I see that `ip_safely` appears now and then in files in the project, but doesn't seem to be explained anywhere.  Can someone explain the convention?